### PR TITLE
Modify rule S2710: LaYC format

### PR DESCRIPTION
**Pull request by maksim-grebeniuk-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

**Review by guillaume-dequenne-sonarsource — APPROVED** [review]
LGTM

File: rules/S2710/python/rule.adoc [hunk 1]
```diff
@@ -1,17 +1,18 @@
+This rule raises an issue when the first parameter of a class method is not an accepted name.
+
 == Why is this an issue?
 
 By convention, the first argument to class methods, i.e. methods decorated with ``++@classmethod++``, is named ``++cls++`` as a representation and a reminder that the argument is the class itself. Name the argument something else, and you stand a good chance of confusing both users and maintainers of the code. It might also indicate that the ``++cls++`` parameter was forgotten, in which case calling the method will most probably fail. This rule also applies to methods ``++__init_subclass__++``, ``++__class_getitem__++`` and ``++__new__++`` as their first argument is always the class instead of "self".
```
Comment (jean-jimbo-sonarsource) [inline rules/S2710/python/rule.adoc:5]: "By convention, the first argument in a class method,..." sounds better

Comment (jean-jimbo-sonarsource) [inline rules/S2710/python/rule.adoc:5]: "Name the argument something else, and you stand a good chance of confusing both users and maintainers of the code" is a bit confrontational in tone. 


"If you were to name the argument something else, you would stand a good chance of confusing both users and maintainers of the code"

**Review by jean-jimbo-sonarsource — APPROVED** [review]

**Review by jean-jimbo-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**